src/sentry/api/endpoints/grouping_level_new_issues.py

```diff
@@ -160,7 +125,7 @@ def _query_snuba(group: Group, unparsed_id: str, offset=None, limit=None):
         # that filtering by timerange (=primary key) is only a little bit
         # faster.
         now = datetime.datetime.now()
-        new_materialized_hash = _get_hash_for_parent_level(group, id, levels_overview)
+        new_materialized_hash = levels_overview.parent_hashes[id]
```

Comment:
This is an unrelated change: I realized that the levels overview now contains the same data that we're computing and caching here.